src/coreclr/vm/jitinterface.h

```diff
@@ -705,6 +706,17 @@ class CEEJitInfo : public CEEInfo
         } CONTRACTL_END;
 
         m_CodeHeader = NULL;
+        
+#ifdef FEATURE_WXORX
```

Comment:
Instead of ifdef, you can make it dynamic: `if (m_CodeHeader != m_CodeHeaderRW)`

Reply:
Done, makes sense